### PR TITLE
Rename NewSchemaLoader to SchemaLoader

### PR DESCRIPTION
**Pull request by swankjesse**
It is the only one now.

### CODE REVIEW

File: wire-library/wire-compiler/src/main/java/com/squareup/wire/JavaFileWriter.kt [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-package com.squareup.wire
-
-import com.squareup.javapoet.JavaFile
-import com.squareup.wire.java.JavaGenerator
-import java.io.IOException
-import java.nio.file.FileSystem
-import java.util.concurrent.Callable
-import java.util.concurrent.ConcurrentLinkedQueue
-
-internal class JavaFileWriter(
```
Comment (swankjesse) [inline wire-library/wire-compiler/src/main/java/com/squareup/wire/JavaFileWriter.kt:25]: Unused!

File: wire-library/wire-compiler/src/main/java/com/squareup/wire/KotlinFileWriter.kt [hunk 2]
```diff
@@ -1,98 +0,0 @@
...
-import com.squareup.wire.kotlin.KotlinGenerator
-import com.squareup.wire.schema.Location
-import com.squareup.wire.schema.Service
-import com.squareup.wire.schema.Type
-import java.io.IOException
-import java.nio.file.FileSystem
-import java.util.concurrent.Callable
-import java.util.concurrent.ConcurrentLinkedQueue
-
-internal class KotlinFileWriter(
```
Comment (swankjesse) [inline wire-library/wire-compiler/src/main/java/com/squareup/wire/KotlinFileWriter.kt:29]: Unused!

File: wire-library/wire-compiler/src/main/java/com/squareup/wire/PendingFileSpec.kt [hunk 3]
```diff
@@ -1,12 +0,0 @@
-package com.squareup.wire
-
-import com.squareup.wire.schema.Service
-import com.squareup.wire.schema.Type
-
-internal sealed class PendingFileSpec
```
Comment (swankjesse) [inline wire-library/wire-compiler/src/main/java/com/squareup/wire/PendingFileSpec.kt:6]: Unused!

**Comment by oldergod** [comment]
:+1:

**Pull request merged**